### DISCUSSION TITLE
Fallback to base locale if page doesn't exist

### DISCUSSION THREAD

**Discussion by tomx-eth** [requarks/wiki#3236] [General]
Suppose the base locale of a wiki is English and the user is browsing in Chinese Traditional.

The user navigates to `/zh-tw/test-page`, but that page does not exist in Chinese Traditional. However `/en/test-page` *does* exist. An admin setting should allow the content from `/en/test-page` to be loaded onto `/zh-tw/test-page` (not a redirect).

This would solve many "This page does not exist yet." errors when translations are not yet complete.

To take it further, the tree navigation could also be mirrored from the base locale.

**Comment by NGPixel** [comment]
This is good idea, please submit to https://feedback.js.wiki/wiki

**Comment by tomx-eth** [comment]
Done!

https://feedback.js.wiki/wiki/p/fallback-to-base-locale-if-page-doesnt-exist
